lib/plugins/deploy-interactive.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
```

Comment:
I'd keep this plugin in `lib/cli/interactive-setup` as it serves only interactive deploy case, and it's a temporary hack, that at some point will be removed

Reply:
:+1: